disruption_py/machine/cmod/physics.py

```diff
@@ -1754,6 +1754,64 @@ def get_sxr_data(params: PhysicsMethodParams):
         sxr = interp1(t_sxr, sxr, params.times)
         return {"sxr": sxr}
 
+    @staticmethod
+    @physics_method(
```

Comment:
you should be able to have these decorator arguments in a single line without black complaining.

Reply:
I tried but it doesn't seem to work.

Reply:
I tried it and it did work. :smile: